big-practice/src/js/dashboard.js

```diff
@@ -37,4 +39,8 @@ const renderTable = () => {
     .catch((error) => alert('An error occurred while getting movie', error));
 };
 
+addBtn.addEventListener('click', () => {
+  modal.classList.add('hide');
```

Comment:
Click add button. However, you add hide class? Something wrong here, pls correct the naming or behavior